governance/related-groups/README.md

```diff
@@ -16,29 +16,29 @@ the work of Security TAG
     - [Storage](https://github.com/cncf/tag-app-delivery) -
 
 * [CNCF TOC](https://github.com/cncf/toc): Technical Oversight Committee
-  * Security TAG members: @justincormack , @lizrice
+<!-- cSpell:ignore justincormack, lizrice-->  * Security TAG members: @justincormack , @lizrice
 
 ## Kubernetes
 
 * [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
-  * Security TAG members: @rficcaglia
+<!-- cSpell:ignore rficcaglia-->  * Security TAG members: @rficcaglia
 
 * [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
   * Security TAG members:
 
 * [Kubernetes SIG-Security](https://github.com/kubernetes/community/tree/master/sig-security)
-  * Security TAG members: @pushkarj
+<!-- cSpell:ignore pushkarj-->  * Security TAG members: @pushkarj
 
 ## Industry
 
 * [Cloud Security Alliance Working Groups](https://cloudsecurityalliance.org/research/working-groups/)
-  * Security TAG members: @achetal01
+<!-- cSpell:ignore achetal01-->  * Security TAG members: @achetal01
 
 * [IEEE 7009](https://standards.ieee.org/develop/project/7009.html): Standard
   for Fail-Safe Design of Autonomous and Semi-Autonomous Systems Working Group
   * Security TAG members:
 
-* [IEEE P1915.1](https://standards.ieee.org/develop/project/1915.1.html):
+* [IEEE P1915.1] :
```

Comment:
I think this change would leave a broken link without the warning, not actually removing the link.

However, I suggest using Internet Archive to refer to the archived version of the page at: https://web.archive.org/web/20180729062624/https://standards.ieee.org/develop/project/1915.1.html

Reply:
My thought was to remove the whole link, but i like the archive link idea, am in favor of using the archive as well (with an add a warning to it that it is using archive).

we could then add an issue to address it more permanently.